### PR TITLE
Scalafix followup:different fix alias on different sub projects with packages ignore support.

### PR DESCRIPTION
**Pull request by He-Pin**
because some subprojects did not enable the MultiNode plugin.

### CODE REVIEW

**Comment by hepin1989** [comment]
without
>[success] Total time: 439 s, completed 2018-12-10 3:45:26

**Review by patriknw — APPROVED** [review]
LGTM, thanks

**Comment by patriknw** [comment]
I tried to add an unsed import in `akka.cluster.sharding.typed.ClusterSharding`
```
import akka.util.Timeout
```

Then I ran `fixall` but the import was not removed. What is expected?

Then I ran `fixall` one more time and then it fails:
```
[error] (akka-camel / Compile / scalafix) scalafix.sbt.InvalidArgument: The semanticdb-scalac compiler plugin is required to run semantic rules like RemoveUnused.
[error] To fix this problem for this sbt shell session, run `scalafixEnable` and try again.
[error] To fix this problem permanently for your build, add the following settings to build.sbt:
[error]   addCompilerPlugin(scalafixSemanticdb)
[error]   scalacOptions += "-Yrangepos"
[error] (akka-multi-node-testkit / Compile / scalafix) scalafix.sbt.InvalidArgument: The semanticdb-scalac compiler plugin is required to run semantic rules like RemoveUnused.
[error] To fix this problem for this sbt shell session, run `scalafixEnable` and try again.
[error] To fix this problem permanently for your build, add the following settings to build.sbt:
[error]   addCompilerPlugin(scalafixSemanticdb)
[error]   scalacOptions += "-Yrangepos"
[error] (akka-slf4j / Compile / scalafix) scalafix.sbt.InvalidArgument: The semanticdb-scalac compiler plugin is required to run semantic rules like RemoveUnused.
[error] To fix this problem for this sbt shell session, run `scalafixEnable` and try again.
[error] To fix this problem permanently for your build, add the following settings to build.sbt:
[error]   addCompilerPlugin(scalafixSemanticdb)
[error]   scalacOptions += "-Yrangepos"
[error] (akka-persistence / Compile / scalafix) scalafix.sbt.InvalidArgument: The semanticdb-scalac compiler plugin is required to run semantic rules like RemoveUnused.
[error] To fix this problem for this sbt shell session, run `scalafixEnable` and try again.
[error] To fix this problem permanently for your build, add the following settings to build.sbt:
[error]   addCompilerPlugin(scalafixSemanticdb)
[error]   scalacOptions += "-Yrangepos"
[error] (akka-persistence-tck / Compile / scalafix) scalafix.sbt.InvalidArgument: The semanticdb-scalac compiler plugin is required to run semantic rules like RemoveUnused.
[error] To fix this problem for this sbt shell session, run `scalafixEnable` and try again.
[error] To fix this problem permanently for your build, add the following settings to build.sbt:
[error]   addCompilerPlugin(scalafixSemanticdb)
[error]   scalacOptions += "-Yrangepos"
```

**Comment by hepin1989** [comment]
@patriknw it currently could only run once for the current session,the behavior was changed for sbt 1.2
`import akka.util.Timeout` is used currently in `akka.cluster.sharding.typed.ClusterSharding`.
I tried the below ,it works

image
after run `fixall`
image

File: build.sbt [hunk 1]
```diff
@@ -4,9 +4,8 @@ enablePlugins(UnidocRoot, TimeStampede, UnidocWithPrValidation, NoPublish, Copyr
   ScalafixIgnoreFilePlugin)
 disablePlugins(MimaPlugin)
 addCommandAlias(
-  name  ="fix",
-  value = ";scalafixEnable;compile:scalafix;test:scalafix;multi-jvm:scalafix;test:compile")
-
+  name  ="fixall",
+  value = ";scalafixEnable;compile:scalafix;test:scalafix;multi-jvm:scalafix;test:compile;reload")
```
Comment (hepin1989) [inline build.sbt]: I added a `;reload`

**Comment by patriknw** [comment]
> Timeout is used currently in akka.cluster.sharding.typed.ClusterSharding

Doh, bad choice of me. Intellij marked it as unused so I thought it was, but it was already imported.

**Comment by hepin1989** [comment]
> > Timeout is used currently in akka.cluster.sharding.typed.ClusterSharding
> 
> Doh, bad choice of me. Intellij marked it as unused so I thought it was, but it was already imported.

I added `reload` to the command alias and try to re-run the `fixall` twice and it works.

**Comment by hepin1989** [comment]
> > Timeout is used currently in akka.cluster.sharding.typed.ClusterSharding
> 
> Doh, bad choice of me. Intellij marked it as unused so I thought it was, but it was already imported.

You should try the latest IDEA version:)

**Review by raboof — APPROVED** [review]
Looks good but curious what those statistics were about?

**Comment by hepin1989** [comment]
With semanticdb on

**Comment by raboof** [comment]
Makes sense! I noticed the `enableScalafix` sbt task will destroy our changes to `scalacOptions`, so it is important to be a bit careful with that - but it is still very useful.

**Comment by hepin1989** [comment]
Ohch should squash it.

**Comment by raboof** [comment]
Oops! Sorry about that - Github UI seems erratic in when they remember the state of the merge button :/.

**Comment by hepin1989** [comment]
It's my problem, Haha

**Pull request merged**